modules/database/src/adapters/mongoose-adapter/index.ts

```diff
@@ -111,8 +111,8 @@ export class MongooseAdapter extends DatabaseAdapter<MongooseSchema> {
         },
         cms: {
           authentication: false,
-          crudOperations: false,
-          enabled: false,
+          crudOperations: {},
```

Comment:
Shouldn't this be `crudOperations: { create: { enabled: false, authenticated: false }, read: ..., update: ..., delete: ... }`?